SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -146,13 +164,20 @@ void SpatialRPCService::PushOverflowedRPCs()
 	{
 		Worker_EntityId EntityId = It.Key().EntityId;
 		ERPCType Type = It.Key().Type;
-		TArray<RPCPayload>& OverflowedRPCArray = It.Value();
+		TArray<PendingRPCPayload>& OverflowedRPCArray = It.Value();
 
 		int NumProcessed = 0;
 		bool bShouldDrop = false;
-		for (RPCPayload& Payload : OverflowedRPCArray)
+		for (PendingRPCPayload& PendingPayload : OverflowedRPCArray)
 		{
-			const EPushRPCResult Result = PushRPCInternal(EntityId, Type, MoveTemp(Payload), false);
+			if (EventTracer != nullptr)
+			{
+				TArray<Trace_SpanId> Causes =
+					PendingPayload.SpanId.IsSet() ? TArray<Trace_SpanId>{ PendingPayload.SpanId.GetValue() } : TArray<Trace_SpanId>{};
+				EventTracer->TraceEvent(FEventRPCRetried(), Causes);
```

Comment:
`Retried` may be a bit misleading because the overflowed RPCs aren't really retried, they're queued on the sender until the buffer empties up enough to send them.